udata/templates/mail/account_deleted.txt

```diff
@@ -0,0 +1,6 @@
+{% extends theme('mail/base.txt') %}
+
+{% block body %}
+{{ _('Your account has now been deleted') }}.
```

Comment:
It would be great to have a recap of what happened for orgs/datasets without any remaining member. Do we allow that by the way?